src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String SOMETHING_WRONG = "Sorry, something went wrong. ";
```

Comment:
Good naming of constant! All CAPS. Also, you could consider removing the empty line above the constant declaration and be consistent with it throughout your code